<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="GLWidget" name="dc_glwidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>351</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSlider" name="voltage_slider">
       <property name="minimum">
        <number>-24</number>
       </property>
       <property name="maximum">
        <number>24</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="voltage_la">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_bt">
       <property name="text">
        <string>LOAD</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="xt_la">
    <property name="geometry">
     <rect>
      <x>371</x>
      <y>163</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="xt_graphic" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>341</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="wt_graphic" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>341</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="wt_la">
    <property name="geometry">
     <rect>
      <x>371</x>
      <y>343</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0.00</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header location="global">glwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
